WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -39,14 +39,15 @@ public class BlockchainController : ControllerBase
 	private static readonly MemoryCacheEntryOptions CacheEntryOptions = new() { AbsoluteExpirationRelativeToNow = TimeSpan.FromSeconds(60) };
 	private static readonly MemoryCacheEntryOptions UnconfirmedTrasanctionChainCacheEntryOptions = new() { AbsoluteExpirationRelativeToNow = TimeSpan.FromSeconds(10) };
 	private static readonly MemoryCacheEntryOptions UnconfirmedTransactionChainItemCacheEntryOptions = new() { AbsoluteExpirationRelativeToNow = TimeSpan.FromSeconds(10) };
-	private static MemoryCacheEntryOptions TransactionCacheOptions { get; } = new MemoryCacheEntryOptions { AbsoluteExpirationRelativeToNow = TimeSpan.FromMinutes(20) };
 
 	public BlockchainController(IMemoryCache memoryCache, Global global)
 	{
 		Cache = new(memoryCache);
 		Global = global;
 	}
 
+	private static MemoryCacheEntryOptions TransactionCacheOptions { get; } = new MemoryCacheEntryOptions { AbsoluteExpirationRelativeToNow = TimeSpan.FromMinutes(20) };
```

Comment:
IMO `readonly` is missing here. Can you use blame and ask the dev who added this line?

Reply:
You added that line:
https://github.com/zkSNACKs/WalletWasabi/commit/aa1bd7c8eca4b9a15f9f50b6e37099b4bf6368d0

Reply:
I made a mistake, can you add `readonly` and move back the line to the original place?

Reply:
Done: ee1521e6206d49f92da753b3301b718bb121623d